CondeNast/launch-vehicle-fbm
# Condé Nast OSS checklist

## Why are we doing this?

Closes #85 

This PR updates the documentation for this SDK to meet the best practices for OSS docs put forth by Condé Nast's Open Source Working Group. The PR relies on @gautamarora's [`opensource-check` tool] as a means to determine if the contents of the repo meet the guidelines. The results of running that tool against this repo before these changes can be reviewed in [this comment].

[`opensource-check` tool]: https://github.com/CondeNast/opensource-check
[this comment]: https://github.com/CondeNast/launch-vehicle-fbm/issues/85#issuecomment-378072970

## Did you document your work?

This PR is 100% documentation related and drive by the aforementioned guidelines and tool.

## How can someone test these changes?

Run the `opensource-check` tool on this repo before and after these changes. The results with these changes:
```
 PASS  Work/opensource-check/__checks__/docs.test.js
  Documentation
    ✓ should have a Readme (5ms)
    ✓ should have a License (1ms)
    ✓ should have a Code of Conduct (2ms)
    ✓ should have Contributing Guidelines (1ms)
    ✓ should have an Issue Template
    ✓ should have a Pull Request Template (1ms)
    ✓ should have a Changelog (1ms)
    ✓ should have Examples (1ms)
    Readme
      ✓ Should have Package Name as Title (1ms)
      ✓ should have an Install section (1ms)
      ✓ should have a Usage section (1ms)

 PASS  Work/opensource-check/__checks__/dev.test.js
  Development
    ✓ should have a package.json (2ms)
    ✓ should have a .gitignore (1ms)
    ✓ should have a Linter (1ms)
    ✓ should have Tests (1ms)
    package.json
      ✓ should have the org namespace, if applicable
      ✓ should have a version (1ms)
      ✓ should have a description
      ✓ should have a main or bin (1ms)
      ✓ should have a repository (1ms)
      ✓ should have contributors (1ms)
      ✓ should have a license

Test Suites: 2 passed, 2 total
Tests:       22 passed, 22 total
Snapshots:   0 total
Time:        0.549s, estimated 1s
Ran all test suites.
```

## What possible risks or adverse effects are there?

None identified.

## What are the follow-up tasks?

None identified.

## Are there any known issues?

None identified.

## Did the test coverage decrease?

Not applicable; no code changes were made in this PR.
